components/case-study/CaseStudyOneBackground.vue

```diff
@@ -0,0 +1,241 @@
+          </p>
+          <p class="sub__section-description">
+            That being said, they must be strategically designed, user-friendly,
+            and optimized to capture crucial information without overwhelming or
+            driving away potential leads.
+          </p>
+          <p class="sub__section-description">
+            As such, it becomes important for any business to recognize this and
+            understand that the key to addressing the lead-generation challenge
+            lies in both creating and optimizing powerful lead-generation forms.
+          </p>
+          <div class="lead-generation-stats d-flex mt-3">
+            <div>
+              <span class="d-block">
+                <span class="sub__section-heading">84</span>
+                %
+              </span>
+              <span class="d-block"
+                >of marketers use form submissions to generate leads.</span
+              >
+            </div>
+          </div>
+        </div>
+
+        <nuxt-img
+          class="sub__section-img img-fluid"
+          src="case-study-1/background/designing-effective-forms.png"
+        ></nuxt-img>
+      </div>
+    </div>
+
+    <div class="my-4 py-5">
+      <div class="d-flex flex-column flex-xl-row gap-112">
+        <div class="d-flex flex-column">
+          <h3 class="sub__section-heading">Formester: A Valuable Tool</h3>
+          <p class="sub__section-description mt-3">
+            In this context, Formester emerges as a valuable tool for businesses
+            looking to optimize their lead-generation forms. Its features, such
+            as simplified form creation, conditional logic, and seamless
+            integration, provide the means to transform forms, from basic data
+            collection tools into powerful lead-capturing and nurturing
+            instruments.  
+          </p>
+          <p class="sub__section-description">
+            Formester empowers businesses to not only create forms but to
+            fine-tune them for success in the competitive digital landscape. 
+          </p>
+          <div class="lead-generation-stats d-flex mt-3">
+            <div>
+              <span class="d-block">
+                <span class="sub__section-heading">50</span>
+                %
+              </span>
+              <span class="d-block"
+                >of marketers say inbound marketing strategies, such as onsite
+                forms, are their primary source of leads</span
+              >
+            </div>
+          </div>
+        </div>
+
+        <nuxt-img
+          class="sub__section-img img-fluid order-xl-first"
+          src="case-study-1/background/formester-a-valuable-tool.png"
+        ></nuxt-img>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  methods: {
+    convertToImageName(text) {
+      return text.toLowerCase().replace(/&/g, 'and').replace(/ /g, '-')
+    },
+  },
+}
+</script>
```

Comment:
Not needed! I have explained the reason here -

![image](https://github.com/formester/formester-site/assets/69912046/d1c40839-f73f-4564-aa83-f369d21eae33)